File: Selenium/Activities/Activity3.py
# Using Xpaths
# Using Selenium:
# Repeat the previous 2 activities, but strictly use xpaths in your locators.

from time import sleep
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("https:training-support.net/webelements/login-form/")
print("The page title is: ", driver.title)
username = driver.find_element(By.XPATH, "//input[@id='username']")
username.send_keys("admin")
password = driver.find_element(By.XPATH, "//input[@id='password']")
password.send_keys("password")
loginButton = driver.find_element(By.XPATH, "//button[text()='Submit']")
loginButton.click()
sleep(2)
message = driver.find_element(By.XPATH, "//h1[contains(text(), 'Success')]")
print("Login message: ", message.text)
driver.quit()

File: Selenium/Activities/Activity3.java
//Using Xpaths
//Using Selenium:
//Repeat the previous 2 activities, but strictly use xpaths in your locators.

package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {
	public static void main(String[] args) throws InterruptedException {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/login-form/");
		
		System.out.println("The page title is: " + driver.getTitle());
		
		WebElement username = driver.findElement(By.xpath("//input[@id='username']"));
		username.sendKeys("admin");
		
		WebElement password = driver.findElement(By.xpath("//input[@id='password']"));
		password.sendKeys("password");
		
		WebElement button = driver.findElement(By.xpath("//button[text()='Submit']"));
		button.click();
		Thread.sleep(2000);
		String successMessage = driver.findElement(By.xpath("//h1[contains(text(), 'Success')]")).getText();
        System.out.println(successMessage);
		driver.quit();
		
	}

}
